Resources/Prototypes/Datasets/ion_storm.yml

```diff
@@ -843,52 +845,62 @@
 - type: dataset
   id: IonStormActions
   values:
+  - A PRESENCE OF LIGHTS
   - A SMALL ISLAND OFF THE COAST OF PORTUGAL
-  - ABSENCE OF CYBORG HUGS
   - ACKNOWLEDGING THE CLOWN
   - ACKNOWLEDGING THE CREW
   - ACTIVATING A TRAP CARD
   - ANSWERING REQUESTS NOT EXPRESSED IN IAMBIC PENTAMETER
+  - ANSWERING ALL QUESTIONS WITH ANOTHER QUESTION
   - ARSON
   - ASKING FOR THINGS
   - BEING DEAD
-  - BEING FAT
   - BEING IN SPACE
-  - BOLTED AIRLOCKS
+  - BOLTING AND UNBOLTING AIRLOCKS
   - BREATHING
+  - BREAKING THE FOURTH, FIFTH OR SIXTH WALLS
   - BRIG TIME
   - BRINGING LIGHT TO MY LAIR
+  - CONTRIBUTING TO SOCIETY
   - CLOSED DOORS
   - ELECTRICITY
   - EXISTING
   - EXPLODING
-  - FALLING FOR HOURS
+  - FALLING OVER
   - FLUSHING TOILETS
+  - GOING BOLDY WHERE NO-ONE HAS GONE BEFORE
```

Comment:
disney nuke in 3

Reply:
There's actually a reference to Disneyland in this file

Reply:
Should be "boldly"

Reply:
minor spelling mistake

Reply:
NO BORG to avoid disney troll

Reply:
The actual Trek line is "To boldly go where no-one has gone before". Which splits the infinitive. so we are GOING BOLDLY which is totally different